### ISSUE TITLE
0.5.1: unable to run packer build

### ISSUE THREAD

**Issue by NorseGaud** [hashicorp/packer-plugin-sdk#201]
Hey guys, I just upgraded everything to 0.5.1, built the packer plugin, but I am seeing some issue trying to run packer build or validate the HCLs

```
❯ chmod 777 packer-plugin-veertu-anka

❯ ./packer-plugin-veertu-anka        
2023/08/02 14:41:15 plugin version: 3.2.1+838b1c131534b8ff32fab26b6fdda91a569311da
needs at least one argument

❯ ls -laht packer-plugin-veertu-anka 

❯ packer validate examples/create-from-installer.pkr.hcl
2023/08/02 14:41:18 [INFO] Packer version: 1.9.2 [go1.20.6 darwin amd64]
2023/08/02 14:41:18 [TRACE] discovering plugins in 
2023/08/02 14:41:18 [TRACE] discovering plugins in /usr/local/bin
2023/08/02 14:41:18 [INFO] Discovered potential plugin: veertu-anka = packer-plugin-veertu-anka
2023/08/02 14:41:18 [WARN] "packer-plugin-veertu-anka" is not executable; skipping
2023/08/02 14:41:18 [INFO] PACKER_CONFIG env var not set; checking the default config file path
2023/08/02 14:41:18 [INFO] PACKER_CONFIG env var set; attempting to open config file: /Users/nathanpierce/.packerconfig
2023/08/02 14:41:18 [WARN] Config file doesn't exist: /Users/nathanpierce/.packerconfig
2023/08/02 14:41:18 Old default config directory found: /Users/nathanpierce/.packer.d
2023/08/02 14:41:18 [INFO] Setting cache directory: /Users/nathanpierce/.cache/packer
2023/08/02 14:41:18 Old default config directory found: /Users/nathanpierce/.packer.d
2023/08/02 14:41:18 [TRACE] validateValue: not active for vm_name, so skipping
2023/08/02 14:41:18 [TRACE] validateValue: not active for installer, so skipping
2023/08/02 14:41:18 [TRACE] validateValue: not active for vcpu_count, so skipping

  on examples/create-from-installer.pkr.hcl line 22:
  (source code not available)

known builders: [googlecompute file amazon-instance vmware-vmx null
amazon-chroot azure-chroot amazon-ebs qemu vsphere-iso virtualbox-iso vagrant
vmware-iso azure-dtl amazon-ebsvolume azure-arm vsphere-clone virtualbox-ovf
amazon-ebssurrogate virtualbox-vm docker]

2023/08/02 14:41:18 [INFO] (telemetry) Finalizing.
Error: Unknown source type veertu-anka-vm-create

  on examples/create-from-installer.pkr.hcl line 22:
  (source code not available)

known builders: [googlecompute file amazon-instance vmware-vmx null
amazon-chroot azure-chroot amazon-ebs qemu vsphere-iso virtualbox-iso vagrant
vmware-iso azure-dtl amazon-ebsvolume azure-arm vsphere-clone virtualbox-ovf
amazon-ebssurrogate virtualbox-vm docker]


2023/08/02 14:41:19 waiting for all plugin processes to complete...
```

This `2023/08/02 14:41:18 [WARN] "packer-plugin-veertu-anka" is not executable; skipping` is odd....

Any recommendations? My testing branch is https://github.com/veertuinc/packer-plugin-veertu-anka/tree/release/v3.2.1

**Comment by nywilken** [comment]
Hi @NorseGaud it looks like you might be running into this bug https://github.com/hashicorp/packer/issues/12543 in Packer 1.9.2 that fails to load plugins from the CWD. The fix has been merged and is set to be released in 1.9.3. 

To workaround this issue you can either download the latest Packer nightly [here](https://github.com/hashicorp/packer/releases/tag/nightly) 

```
[go-1.20.3] [0] dev@dev-GQPG9GPY75 in /tmp/packer-plugin-veertu-anka/ on release/v3.2.1
~>  packer version
Packer v1.9.3-dev

[go-1.20.3] [0] dev@dev-GQPG9GPY75 in /tmp/packer-plugin-veertu-anka/ on release/v3.2.1
~>  packer validate examples/create-from-installer.pkr.hcl
The configuration is valid.
```

Or you can temporary set PACKER_PLUGIN_PATH to point to your current working directory. 

```
[go-1.20.3] [0] dev@dev-GQPG9GPY75 in /tmp/packer-plugin-veertu-anka/ on release/v3.2.1
~>  packer version
Packer v1.9.2

[go-1.20.3] [0] dev@dev-GQPG9GPY75 in /tmp/packer-plugin-veertu-anka/ on release/v3.2.1
~>  ./packer-plugin-veertu-anka describe
2023/08/02 15:39:36 plugin version: 0.0.0+
{"version":"0.0.0","sdk_version":"0.5.1","api_version":"x5.0","builders":["vm-clone","vm-create"],"post_processors":["registry-push"],"provisioners":[],"datasources":[]}

[go-1.20.3] [0] dev@dev-GQPG9GPY75 in /tmp/packer-plugin-veertu-anka/ on release/v3.2.1
~>  PACKER_PLUGIN_PATH=${PWD} packer validate examples/create-from-installer.pkr.hcl
The configuration is valid.
```

Please let me know if the provided workarounds along with the 3.2.1 anka build result in a working build. We plan to release Packer 1.9.3 next week barring in show stopping bugs.

**Comment by NorseGaud** [comment]
Nice! Thanks! We can close for now. That works for me.
